docs/docs/architecture.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Architecture
+---
+# Eraser architecture
+
```

Comment:
can we add a brief description here? we can take from google docs probably

let's call out metrics are not available in v0.4